docs/rules/max-classes-per-file/index.html

```diff
@@ -0,0 +1,56 @@
+---
```

Comment:
FYI, it's a little confusing that we need to run the `validate` NPM script, but then know not to check in certain files that it generates.